src/select2.component.ts

```diff
@@ -148,6 +148,29 @@ export class Select2Component implements AfterViewInit, OnChanges, OnDestroy, On
             this.renderer.setElementProperty(this.selector.nativeElement, 'disabled', this.disabled);
         }
     }
+     
+    private setElementValue ( newValue ) {
```

Comment:
I've found that the function always receives string values from user interactions, both for multiple and single selects. In the case of multiple selects that's a comma separated string of multiple values. I'm not sure if this is coming from the native select2 lib or the browser.